### PR TITLE
Add e2e tests and refactoring on back-end fixtures

### PR DESCRIPTION
**Pull request by darkweak**

### CODE REVIEW

**Comment by jenaye** [comment]
thanks to @darkweak for this awesome work

and  @siroc for creating new English and French vulnerabilities

image

**Review by jenaye — APPROVED** [review]

File: api/fixtures/vuln.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  sti (extends vulnerability):
+    translations:
+      - '@SSTI_fr'
+      - '@SSTI_en'
+
+  osinjection (extends vulnerability):
+    translations:
+      - '@OSinjection_fr'
+      - '@OSinjection_fr'
```
Comment (darkweak) [inline api/fixtures/vuln.yaml:46]: Missing NL @jenaye

File: api/fixtures/point.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+App\Entity\PositivePoint:
+  positive_point_cloisonnement:
+    name: Cloisonnement
```
Comment (darkweak) [inline api/fixtures/point.yaml:3]: That's not an english word

Comment (darkweak) [reply api/fixtures/point.yaml:3]: @jenaye

**Pull request merged**